test/public/lhcFills/detail.test.js

```diff
@@ -34,11 +34,34 @@ module.exports = () => {
         expect(stableBeamBadge).to.be.not.null;
     });
 
+    it('should successfully display runs statistics', async () => {
+        const statistics = await page.$('#statistics');
+        expect(statistics).to.be.not.null;
+        const statisticsContent = await page.$eval('#statistics', (element) => element.innerText);
+        expect(statisticsContent).to.include('Over 2 minutes');
+        expect(statisticsContent).to.include('Under 2 minutes');
+        expect(statisticsContent).to.include('Per quality');
+        expect(statisticsContent).to.include('Per detectors');
+    });
+
     it('should successfully display runs related to the fill', async () => {
         const runsTable = await page.$$('#runs tbody tr');
         expect(runsTable.length).to.be.greaterThan(0);
     });
 
+    it('should successfully switch to physics runs only', async () => {
```

Comment:
Is this test still applicable now that the tabs of run definitions was removed?

Reply:
Of course not, my bad